addons/web/static/src/core/model_field_selector/model_field_selector_popover.js

```diff
@@ -15,6 +16,7 @@ export class ModelFieldSelectorPopover extends Component {
         this.fieldKeys = [];
         this.searchValue = "";
         this.fullFieldName = this.fieldNameChain.join(".");
+        !this.env.isSmall && useAutofocus();
```

Comment:
Might personal opinion, but since you call a function, I'd rather see a if statement. I'm ok with the && when the value is assigned, but less a fan with such usage